WordPress/Classes/ViewRelated/Stats/Shared Views/Post Stats/PostStatsTableViewController.swift

```diff
@@ -68,3 +74,15 @@ private extension PostStatsTableViewController {
     }
 
 }
+
+// MARK: - PostStatsDelegate Methods
+
+extension PostStatsTableViewController: PostStatsDelegate {
+
+    func displayWebViewWithURL(_ url: URL) {
+        let webViewController = WebViewControllerFactory.controllerAuthenticatedWithDefaultAccount(url: url)
+        let navController = UINavigationController.init(rootViewController: webViewController)
+        present(navController, animated: true, completion: nil)
```

Comment:
Fun fact: the completion handler of `present(_:animated:completion:)` has a default parameter value of `nil`. So, you could delete that if you were so inclined.

Reply:
Ah, cool. Thanks! I took the liberty of removing that param from all the Stats `displayWebViewWithURL` methods.